src/scss/components/overlays/_touch-control-overlay.scss

```diff
@@ -23,6 +23,7 @@
   opacity: 1;
   position: absolute;
   text-shadow: $color-shadow-seek-label;
+  z-index: 1;
```

Comment:
The `::before` element is above the label. To fix this we apply the z-index of 1 here.